keyboards/satan/keymaps/olligranlund_iso/keymap.c

```diff
@@ -48,6 +48,10 @@
 #define CALC KC_CALC  // Open default calculator app
 #define MYCM KC_MYCM  // Open default file manager
 
+// increase readability
+#define _______  KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```